packages/web/src/app/clubs/page.tsx

```diff
@@ -1,11 +1,145 @@
 "use client";
 
+import { useState } from "react";
+
+import Card from "@sparcs-clubs/web/common/components/Card";
+import PageTitle from "@sparcs-clubs/web/common/components/PageTitle";
+import SectionTitle from "@sparcs-clubs/web/common/components/SectionTitle";
 import { UseClientProvider } from "@sparcs-clubs/web/common/providers/UseClientProvider";
+import styled from "styled-components";
+import Tag from "@sparcs-clubs/web/common/components/Tag";
+
+import clubsData from "./data/clubs";
+
+const EmptyGap = styled.div<{ height: number }>`
+  height: ${props => props.height}px;
+`;
+
+const SectionTitleContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const OpenToggleButton = styled.div`
+  font-size: 15px;
+  text-decoration-line: underline;
+  color: ${({ theme }) => theme.colors.GRAY};
+`;
+
+const CardsContainer = styled.div`
+  width: 100%;
+  margin-top: 10px;
+  padding-left: 25px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 14px;
+`;
+
+const CardNameContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+const CardName = styled.div`
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
+  font-size: 20px;
+`;
+
+const CardMembersContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const CardMembers = styled.div`
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.REGULAR};
+  font-size: 14px;
+`;
+
+const CardPresident = styled.div`
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.REGULAR};
+  font-size: 16px;
+`;
+
+const CardCharacteristic = styled.div`
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.REGULAR};
+  font-size: 16px;
+`;
+
+const clubTypeHandler = (clubType: string) => {
+  switch (clubType) {
+    case "상임동아리":
+      return "GREEN";
+    case "정동아리":
+      return "BLUE";
+    default:
+      return "ORANGE";
+  }
+};
+
+const Clubs = () => {
+  const [openToggle, setOpenToggle] = useState<boolean>(false);
```

Comment:
```suggestion
  const [openToggle, setOpenToggle] = useState(false);
```
이 경우에 openToggle은 initialize한 값인 false에 따라 이미 자동으로 boolean으로 타입이 정해지기 때문에 `<boolean>`을 써주지 않아도 됩니다!

useState에서 이러한 타이핑을 해주면 좋은 경우는 initialize한 값만으로 정확한 타이핑이 되지 않을 경우인데,
(1) 타입이 확장되는 케이스 
ex) `const stringOrNull = useState<string | null>(null);` -> 이 경우에는 타입 명시를 안해주면 null이라고 생각함!
(2) 타입이 축소되는 케이스
ex) `const clubType = useState<"상임동아리" | "정동아리" | "가동아리">("상임동아리");` -> 이 경우에는 타입 명시를 안해주면 (모든) string이라고 생각함!